authorization.md

```diff
@@ -440,3 +441,22 @@ Pragma: no-cache
   "scope": "system/*.read system/CommunicationRequest.write"
 }
 ```
+
+## Presenting an Access Token to FHIR API
+
+With a valid access token, a client MAY issue a FHIR API call to a FHIR resource server or other appropriate endpoint. The request MUST include an ```Authorization``` header that presents the ```access_token``` as a “Bearer” token:
+
+```
+Authorization: Bearer {{access_token}}  
+```
+
+[where {{access_token}} is replaced with the actual token value]
+
+**Example Request**
+
+```
+GET https://system/*.read system/CommunicationRequest.write
```

Comment:
```suggestion
GET https://ehr.example.org/metadata
```

Reply:
Quick fix for this example, since the `system/*.read` wouldn't show up in a real URL.